config/locales/nav.en.yml

```diff
@@ -5,7 +5,7 @@ en:
       incoming_cases: New cases
       my_open_cases: My open cases
       open_cases: All open cases
-      case_retention: Case Retention
+      rrd_pending: RRD Pending
```

Comment:
This is great thanks, I completely forgot about this env file entry when I added that rrd heading in (which is silly really since I added the case_retention one 😂 )